tests/integration_tests/functional/test_max_devices.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests scenario for adding the maximum number of devices to a microVM."""
+
+import platform
+import pytest
+import host_tools.network as net_tools
+
+# IRQs are available from 5 to 23, so the maximum number of devices
+# supported at the same time is 19.
+MAX_DEVICES_ATTACHED = 19
```

Comment:
In order to avoid confusion later on just set this to 18 and use +1 where you want to exceed limit.

Reply:
Please see the comments above. The maximum number of attached devices is 19, not 18.